symd.py

```diff
@@ -16,6 +16,12 @@
 import re
 import json
 
+"""
+Code fix to avoid errors while creating the plot. (with option --graph)
+"""
+from pylab import *
```

Comment:
What functions are required from this module? Can we do a more targeted import rather than everything?

Reply:
When we set matplotlib to non-interactive mode with command "plt.interactive(False)", the "plt" is not definend.
We could uncomment the line: "import matplotlib.pyplot as plt" to make error "plt is not defined" dissappear.